test/runtests.jl

```diff
@@ -20,6 +20,12 @@ end
 
 @testset for fixture in readdir(TMP_FIXTURES)
     results = test_runner("", "$TMP_FIXTURES/$fixture/")
+    if endswith(fixture, "syntax_error")
+        # Substitute prefix with local file name in stack trace
+        HOME_PREFIX = joinpath("~", relpath(pkgdir(ExercismTestReports), homedir()))
+        TEST_PREFIX = "~/projects/exercism/repos/julia-test-runner"
+        results = replace(results, HOME_PREFIX => TEST_PREFIX)
```

Comment:
This will only work if the pkgdir is in the home directory and won't work at all on windows, though it should be harmless...

Let's merge it, it's better than the status quo.

Reply:
You're right. One could use `Base.Filesystem.contractuser` here which would make it independent of the operating system (identity function on `Sys.windows`) and where the package is installed (function checks whether it starts with home).

Is there a way to re-open and modify this pull request or shall I create a new pull request for that?